CHANGES.md

```diff
@@ -1,6 +1,329 @@
+* improved the documentation with respect to profiling  
+  (see https://chapel-lang.org/docs/1.33/usingchapel/building.html#makefile-options  
+   and https://chapel-lang.org/docs/1.33/developer/bestPractices/GeneratedCode.html#profiling-the-generated-code)
+* updated 'mason' docs and example to show required use of `--` before `-M`  
+  (see https://chapel-lang.org/docs/1.33/tools/mason/guide/buildinglargerpackages.html)
+* added missing documentation for symbols in stable standard modules
+
+Example Codes
+-------------
+* removed LCALS & PTRANS from the benchmarks directory due to lack of attention
+* updated example codes with respect to changes in the language and libraries
+
+Syntax Highlighting
+-------------------
+* added missing keywords to the syntax highlighter for 'vim'
+
+Configuration / Build / Packaging Changes
+-----------------------------------------
+* made `make install` create the installation directory rather than `configure`
+* refreshed the sample installation commands in the prerequisites docs  
+  (see https://chapel-lang.org/docs/1.33/usingchapel/prereqs.html#installation)
+
+Portability / Platform-specific Improvements
+--------------------------------------------
+* added support for system installations of LLVM 16  
+  (see https://chapel-lang.org/docs/1.33/usingchapel/prereqs.html)
+* addressed a problem building the compiler on recent versions of FreeBSD
+
+Compiler Improvements
+---------------------
+* started ignoring `when` blocks known to be false when inferring return types
+
+Compiler Flags
+--------------
+* added `--[no-]detailed-errors` to request error messages with more detail  
+  (see https://chapel-lang.org/docs/1.33/usingchapel/man.html)
+* added `--[no-]const-arg-checks` to check for potential `const` violations  
+  (see https://chapel-lang.org/docs/1.33/usingchapel/man.html)
+* renamed the `CHPL_NO_*` env vars to make their behavior more predictable  
+  (e.g. `CHPL_NO_CHECKS` is deprecated in favor of `CHPL_CHECKS`)  
+  (see `chpl --help-env` for environment variable names)
+
+Launchers
+---------
+* added co-locale support to `slurm-gasnetrun_*` and `gasnetrun_*` launchers
+
+Error Messages / Semantic Checks
+--------------------------------
+* added an error when two files define a top-level module with the same name
+* added an error for `ref` declarations with incorrect declared types
+* added a warning for cases where symbol shadowing might be surprising  
+  (e.g., if `M` and `N` both define `x`, `use M; public use N; f(x);` warns)
+* added a compiler error when casting to a generic subclass in unsupported ways
+* added an unstable warning when `const` arguments are indirectly modified  
+  (see https://chapel-lang.org/docs/1.33/language/spec/procedures.html#the-const-intent)
+* fixed the unstable warning for `gpuUseStreamPerTask` to use the correct name
+* improved the output format of `--detailed-errors` since its dev-only form
```

Comment:
This line doesn't scan quite right to me.

Reply:
Moved it down and reworded it, which I think makes this much more satisfactory.